build-custom-resource-handlers.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
I wonder if a better way to organize this code is to use a lerna project with a private module that contains the runtime code (see [cdk-tweet-queue](https://github.com/eladb/cdk-tweet-queue) as an example). It will make it much easier to control reuse, use dependencies (fs-extra is called for), test in isolation, distinguish internal/external etc.

At the same time, you could then put all the general purpose Lambda-CFN stuff into yet-another module in the lerna repo and publish it separately for the world to consume and reuse, or consume something like [cfn-lambda](https://www.npmjs.com/package/cfn-lambda) - why reinvent the wheel?

Reply:
Because this change already blew *way* beyond what was strictly necessary.